formal/tla/lifecycle/results.md

```
@@ -0,0 +1,918 @@
+(`\A o \in RawInputs : objContent[o] = InitContent(o)`) to pin the
+assumption so a future edit cannot silently add a raw-input mutation without
+tripping a checked invariant. Named to read as an environmental assumption
+being asserted, not a protocol property being proven (unlike the thirteen
+load-bearing invariants above, it does not follow from anything else in the
+model; it is taken as given). Checked in `smoke.cfg` and `exhaustive.cfg`.
+
+Non-vacuity: a scratch mutant (`counterexamples/raw-input-immutable-mutant.md`)
+adds a `MutateRawInput` action, disjuncted into `Next`, that clears
+`objContent["raw1"]`. Against `smoke.cfg` with the new invariant in the list:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)